test/automation/cluster_props.sh

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export ACCUMULO_REPO=https://github.com/apache/accumulo.git
+export ACCUMULO_BRANCH=master
+
+export ACCUMULO_TESTING_REPO=https://github.com/apache/accumulo-testing.git
+export ACCUMULO_TESTING_BRANCH=master
+
+export FLUO_MUCHOS_REPO=https://github.com/apache/fluo-muchos
+export FLUO_MUCHOS_BRANCH=master
+
+# path/to/muchos.props
+export MUCHOS_PROPS=
```

Comment:
File looks good, just need a default value for MUCHOS_PROPS

```suggestion
export MUCHOS_PROPS=${MUCHOS_PROPS:-/path/to/muchos/props}
```
If you set it using this syntax the value can come from your env if set, otherwise default to a non existent path.  This way a user can just set the value in the env or modify this value.

Reply:
Made those changes, thanks for the feedback.